docs/References.bib

```diff
@@ -308,7 +308,7 @@ @article{Minerbo1978
   year    = "1978",
   issn    = "0022-4073",
   doi     = "10.1016/0022-4073(78)90024-9",
-  url     = "http://www.sciencedirect.com/science/article/pii/0022407378900249"
+  url     = "https://doi.pangaea.de/10.1016/0022-4073(78)90024-9"
```

Comment:
I think we should go through `doi.org`

Reply:
I tried to use an alternative that actually guaranties a certain level of information privacy, but I will do as you wish.

Reply:
If we want to make such I change I think it should be discussed as an issue first and then done to all of them at once :)